tensorboard/plugins/scalar/summary_v2.py

```diff
@@ -30,6 +30,34 @@
 def scalar(name, data, step=None, description=None):
     """Write a scalar summary.
 
+    See also `tf.summary.image`, `tf.summary.histogram`, `tf.summary.SummaryWriter`.
+
+    Logs simple numeric values for later analysis in TensorBoard.  Each summary
+    point is associated with an integral `step` value. This enables the incremental
+    logging of time series data.  A common usage of this API is to log loss during
+    training to produce a loss curve.
+
+    For example:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logdir')
+    with test_summary_writer.as_default():
+        tf.summary.scalar('loss', 0.345, step=1)
+        tf.summary.scalar('loss', 0.234, step=2)
+        tf.summary.scalar('loss', 0.123, step=3)
+    ```
+
+    Multiple independent time series may be logged by giving each series a unique
+    `name` value.
+
+    See https://www.tensorflow.org/tensorboard/get_started for more examples
+    of effective usage of `tf.summary.scalar`.
+
+    There is no constraint to prevent the user of this API from logging multiple
```

Comment:
This comment seems potentially a little confusing?  Maybe worth framing this as something a little more explicit:

"In general, this API expects that data points are logged with a monotonically increasing step value.  Duplicate points for a single step or points logged out of order by step are not guaranteed to display as desired in TensorBoard."